imod_coupler/logging/exchange_collector.py

```diff
@@ -0,0 +1,107 @@
+import abc
+import os
+from pathlib import Path
+from typing import Any, List, Optional
+
+import netCDF4 as nc
+import numpy as np
+import tomli
+from numpy.typing import NDArray
+from typing_extensions import Self
+
+
+class AbstractExchange(abc.ABC):
+    def __init__(self, name: str):
```

Comment:
I *think* you typically don't add `__init__` to abstract classes, but maybe good to double check.

Reply:
removed the constructor. tests still pass.